packages/client/package.json

```diff
@@ -34,7 +34,7 @@
     "honeycomb-grid": "^3.1.3",
     "jest-mock-console": "^1.0.0",
     "json-metaschema": "^1.2.0",
-    "leaflet": "^1.6.0",
+    "leaflet": "^1.8.0",
```

Comment:
@lilitkarapetyan  - I _think_ we only refer to `leaflet` from the `packages/components` package.   Do we need these 4 `leaflet` requirements in `packages/client`